FordLabs/retroquest
# Enable csrf protection

## Overview
Enables CSRF protections on the backend

## Testing Instructions
- Run the backend and frontend locally and ensure all API calls still behave as expected
- Run the application in a deployed environment and ensure all API calls still behave as expected
